gio/src/task.rs

```diff
@@ -60,11 +61,42 @@ impl Task {
         }
     }
 
+    #[doc(alias = "g_task_return_value")]
+    pub fn return_value<V: ValueType>(&self, result: &V) {
```

Comment:
Maybe even go with `return_result<V: ValueType>(&self, result: Result<&V, glib:Error>)` and remove `return_error`?